packages/backend/src/payment-method/ilp/service.ts

```diff
@@ -327,6 +328,7 @@ async function pay(
       },
       'ILP payment completed'
     )
+    return { receive: receipt.amountDelivered, debit: finalDebitAmount }
```

Comment:
I think we should leave a comment here explaining the behaviour why not the receipt.amountSent